### PR TITLE
Add GEstimator manifest

### PR DESCRIPTION
**Pull request by manuvarkey**
Homepage: https://manuvarkey.github.io/GEstimator/
Sources: https://github.com/manuvarkey/gestimator/

GEstimator is a civil estimation software for preparing cost estimates of civil/electrical works along with detailed rate analysis. It supports multiple user databases and comes bundled with DSR 2016 (Civil) and DSR 2016 (E and M). Final estimate including the schedule of rates, schedule of resources, resource usage and analysis of rates for various items of work can be exported to a spreadsheet for further manipulation.

### CODE REVIEW

File: org.kavil.gestimator.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "org.kavil.gestimator",
```
Comment (nedrichards) [inline org.kavil.gestimator.json]: Is this app-id correct? There doesn't seem to be a domain registered for kavil.org

File: org.kavil.gestimator.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
```
Comment (nedrichards) [inline org.kavil.gestimator.json]: Is there a particular reason 3.32 doesn't work? Whilst the base for 3.30 is still in security support it doesn't really get fixes from the GNOME project any more.

File: org.kavil.gestimator.json (hunk 1 above)
Comment (manuvarkey) [reply org.kavil.gestimator.json]: Sorry the domain given is not registered. Have now registered kavilgroup.com. Will resubmit merge request after making correction to the app id as com.kavilgroup.gestimator. Sorry for the inconvenience caused. I will close this pull request now.

File: org.kavil.gestimator.json (hunk 2 above)
Comment (manuvarkey) [reply org.kavil.gestimator.json]: App works fine under 3.32. Will re-base to 3.32.

**Comment by manuvarkey** [comment]
Wrong app-id given will resubmit pull request. Sorry for the inconvenience caused.

File: org.kavil.gestimator.json (hunk 1 above)
Comment (nedrichards) [reply org.kavil.gestimator.json]: You didn't need to close the PR, you can just push a new commit if you wish

**Pull request closed without merging**